### PR TITLE
Added lbl:bbox fields to SF and Tokyo

### LINKED ISSUE #361: Add sample custom label bounding boxes to SF and Tokyo
Picking up from @orangejulius suggestion in https://github.com/pelias/pelias/issues/370, let's add samples in WOF for San Francisco locality and Tokyo locality & maybe region, too, setting a custom "label bounding box" that is tight on the main land mass (e.g.: excluding the [Farallon Islands](https://en.wikipedia.org/wiki/Farallon_Islands) for SF).

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #361 

For the [San Francisco](https://whosonfirst.mapzen.com/spelunker/id/85922583/#10/37.7850/-122.7278) record:

* Added `lbl:bbox` field with new bounding box of contiguous SF geometry (does not include Treasure Island or Farallon Islands). This bounding box was taken from the `85922583-alt-mapzen.geojson` file.
* Updated `lastmodified` date.

For the [Tokyo](https://whosonfirst.mapzen.com/spelunker/id/85672817/#5/30.218/146.454) region record:
* Added `lbl:bbox` field with new bounding box of mainland Tokyo area. The bounding box was taken from the [Tokyo locality](https://whosonfirst.mapzen.com/spelunker/id/102031307/#10/35.6734/139.7133) record.
* Updated `lastmodified` date.

### CODE REVIEW

File: data/859/225/83/85922583.geojson [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "gn:longitude":-122.41942,
     "gn:population":805235,
     "iso:country":"US",
+    "lbl:bbox":"-122.51489,37.70808,-122.35698,37.81157",
```
Comment (nvkelso) [inline data/859/225/83/85922583.geojson]: Please also include Treasure Island in the upper-right corner (`-122.35698,37.81157`).

File: data/856/728/17/85672817.geojson [hunk 2]
```diff
@@ -10,6 +10,7 @@
     "geom:longitude":139.578699,
     "gn:population":12445327,
     "iso:country":"JP",
+    "lbl:bbox":"139.279417452,35.5330801096,140.147167867,35.813461192",
```
Comment (nvkelso) [inline data/856/728/17/85672817.geojson]: Sure.

File: data/859/225/83/85922583.geojson (hunk 1 above)
Comment (stepps00) [inline data/859/225/83/85922583.geojson]: Newest commit updated the upper-right corner values, though the Bayview/Hunters Point extends further east than Treasure Island, so only 1/4 lat/long values was updated.

**Comment by stepps00** [comment]
[Newest commit](c872fb6) updated the `lbl:bbox` values for San Francisco.

**Comment by nvkelso** [comment]
LGTM :)

**Comment by nvkelso** [comment]
@thisisaaronland Should we also add

`src:lbl:bbox=mapzen`?

**Comment by thisisaaronland** [comment]
@stepps00 this looks fine and I will merge - can you please update https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/lbl.md as necessary 

@nvkelso I don't think so; I think that can be assumed.

**Pull request merged**